.github/ISSUE_TEMPLATE/a-breaking-change.yml

```diff
@@ -21,10 +21,6 @@ body:
       label: Version
       description: What version of .NET introduced the breaking change?
       options:
-        - .NET 8 Preview 6
```

Comment:
@gewarren LMK if this is correct. Should future updates be added? Do we still get breaking changes on .NET 8?

Reply:
Yes, it's fine to remove these, and you could also add a few more preview versions for .NET 9. Thanks!